index.js

```diff
@@ -607,9 +607,8 @@ WuiDom.prototype.delClassNames = function (classNames) {
  * @param {Boolean} [shouldAdd]
  */
 WuiDom.prototype.toggleClassNames = function (classNames, shouldAdd) {
-	classNames = joinArgumentsAsClassNames('', classNames);
-
-	if (arguments.length > 1) {
+	if (shouldAdd !== undefined) {
+		classNames = joinArgumentsAsClassNames('', classNames);
```

Comment:
Ah, it's my bad. I think all functions no longer take strings, but only arrays. So your current approach is perfectly valid. I actually thought joinArgumentsAsClassNames would allow the 2nd argument to be a string *or* an array of strings. But that's not the case. I'm sure this strictness has good performance benefits, and I'm okay with sticking for it. In any case, it's not the purpose of your PR to change *that* behavior. So... long story short: LGTM :)